README.md

```diff
@@ -22,6 +22,50 @@ And this will be your templates after you use Mustache:
 
 !['stache](https://cloud.githubusercontent.com/assets/288977/8779228/a3cf700e-2f02-11e5-869a-300312fb7a00.gif)
 
+## Install
+
+You can get Mustache via npm.
+
+```bash
+$ npm install mustache --save
+```
+or install with bower:
+
+```bash
+$ bower install --save mustache
+```
+
+## Command line tool
+
+mustache.js is shipped with a node based command line tool. It might be installed as a global tool on your computer to render a mustache template of some kind
+
+```bash
+$ npm install -g mustache
+
+$ mustache dataView.json myTemplate.mustache > output.html
+```
+
+also supports stdin.
+
+```bash
+$ cat mustache dataView.json | mustache - myTemplate.mustache > output.html
```

Comment:
I'm not sure how this slipped through, but it should be `cat dataView.json` instead of `cat mustache dataView.json`.